assets/common.sh

```diff
@@ -6,7 +6,7 @@ parse_source_config() {
 
 get_listing() {
   GLOB=$(echo "$REGEXP" | sed 's/(.*)/*/')
-  LISTING=$(lftp -c "open $URL; cls -1 $GLOB")
+  LISTING=$(lftp -c "open $URL; set cmd:cls-default ''; cls -1 $GLOB")
```

Comment:
I'm not sure what other defaults might have been there before we reset it. Removing `-F` looks correct though.